CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 
 ### Deleted
+* [ALFREDAPI-447](https://xenitsupport.jira.com/browse/ALFREDAPI-447): Removed LuceneNodeVisitor
```

Comment:
Maybe clarify that this is an unused internal class, users of the API won't be affected.

Reply:
@rschev I've added more information to the changelog entry